calculate.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require_once 'database/database.php';
+
+$wyniki = $pdo->query('SELECT * FROM leki')->fetchAll(PDO::FETCH_ASSOC);
+
+foreach ($wyniki as $wynik)
+
+    {
+        $dzisiejsza_data = date("Y-m-d");
+        $data_dodania = $wynik['Data'];
+        $zapotrzebowanie = $wynik['Zapotrzebowanie'];
+
+        $datetime1 = new DateTime($data_dodania);
+        $datetime2 = new DateTime($dzisiejsza_data);
+        $interval = $datetime1->diff($datetime2);
+        $pozostalo = (int)($interval->format('%R%a days')) * ($zapotrzebowanie);
```

Comment:
Jak usuniesz "days" to nie musisz castować do inta.

Reply:
a no tak, wziąłem to na pałę z dokumentacji i myślałem, że days jest częścią formatowania a nie opisem ;]